tests/disabled/partest/run/StringConcat.scala

```diff
@@ -0,0 +1,86 @@
+// java: -Xss128M
```

Comment:
I'd appreciate some guidance for how this sort of test is supposed to be written in Dotty.

Reply:
Is this the stacksize needed to compile the test? I don't think we have any way to customize that currently unfortunately since we always compile tests in the same JVM.

Reply:
Yes. The default stack is insufficient when type checking the massive string concatenation expression (which is sort of reasonable if the type checker is doing some traversal of the AST, since that AST is going to be quite deep).

Reply:
Could the expression be rewritten as a balanced binary-tree to decrease the stack-size (`((a+ b) + (c +d) + (...)` or is the shape important for the test?

Reply:
Shape doesn't matter. This might just work... I'll try.